mysql-test/suite/galera/t/pxc_start_replica_during_sst_donate.test

```diff
@@ -0,0 +1,112 @@
+# ==== Purpose ====
+#
+# This test verifies that starting async replication while proccessing SST
+# request are handled properly by the server.
+#
+# ==== Implementation ====
+#
+# 1. Create a two node cluster.
+# 2. Shutdown node2 and remove the grastate.dat file.
+# 3. Execute some transactions on node1 so that node2 forces an SST when it
+#    joins next time.
+# 4. Add a debug point on node1 to halt just before donating state transfer.
+# 5. Start node2 in background and wait till node1 stops before donating SST
+# 6. Configure replication parameters and execute START REPLICA.
+# 7. Shutdown node2 started as a background process and restart
+# 8. Cleanup
+#
+# ==== References ====
+#
+# PXC-4219: Deadlock with concurrent START SLAVE and backup using PXB
+#
+
+
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+
+--echo #
+--echo # 1. Create a two node cluster.
+--source include/galera_cluster.inc
+
+--echo #
+--echo # 2. Shutdown node2 and remove the grastate.dat file.
+--connection node_2
+--echo [connection node_2]
+--source include/shutdown_mysqld.inc
+--remove_file $MYSQLTEST_VARDIR/mysqld.2/data/grastate.dat
+
+# Wait until the cluster size is updated on node1.
+--connection node_1
+--echo [connection node_1]
+--let $wait_condition = SELECT VARIABLE_VALUE = 1 FROM performance_schema.global_status WHERE VARIABLE_NAME = 'wsrep_cluster_size';
+--source include/wait_condition.inc
+
+--echo #
+--echo # 3. Execute some transactions on node1 so that node2 forces an SST when it
+--echo #    joins next time.
+CREATE TABLE t1 (f1 INTEGER PRIMARY KEY) ENGINE=InnoDB;
+INSERT INTO t1 VALUES(1);
+INSERT INTO t1 VALUES(2);
+
+--echo #
+--echo # 4. Add a debug point on node1 to halt just before donating state transfer.
+--let $debug_point= halt_before_sst_donate
```

Comment:
The whole file: Inconsistent spaces around `=`

Reply:
Done. Used `= ` for `--let` mtr assignments and `  =  ` for SQL statements. And commit message was also updated.